axes/signals.py

```diff
@@ -17,7 +17,7 @@
 log = getLogger(settings.AXES_LOGGER)
 
 
-user_locked_out = Signal(providing_args=["request", "username", "ip_address"])
+user_locked_out = Signal()
```

Comment:
You could add a comment so the information from `providing_args` is not lost forever. :)

Reply:
Done. :)

I would have liked to add a bit about the signal in the docs (it's not mentioned anywhere), but I wasn't sure where it would belong.

Reply:
Nice! But I think a comment should be sufficient for now anyway.